content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -74,82 +24,134 @@ pulling to/from both Git and DVC remotes in the case of experiments.
           │      dvc exp pull     │
           ▼                       ▼
   ┌─────────────────┐    ┌────────────────┐
-  │   Code and      │    │                │
-  │   metafiles     │    │   Cached data  │  Local project
+  │    Code and     │    │      Data      │
+  │    metafiles    │    │    (cached)    │  Local project
   └─────────────────┘    └────────────────┘
 ```
 
-[dvc experiments]: /doc/user-guide/experiment-management/experiments-overview
```

Comment:
BTW this old link was broken it seems, and there's more instances throughout the docs...